Cubical/Data/Group/Base.agda

```diff
@@ -21,19 +25,81 @@ record isGroup {ℓ} (A : Type ℓ) : Type ℓ where
     rCancel : ∀ a → comp a (inv a) ≡ id
 
 record Group ℓ : Type (ℓ-suc ℓ) where
+  no-eta-equality
   constructor group
   field
     type : Type ℓ
     setStruc : isSet type
     groupStruc : isGroup type
 
 isMorph : ∀ {ℓ ℓ'} (G : Group ℓ) (H : Group ℓ') → (f : (Group.type G → Group.type H)) → Type (ℓ-max ℓ ℓ')
```

Comment:
I don't dare to change this now that all the new crazy stuff about structures that's been added. But maybe this could be added as a lemma in the Morph record in the future. I'll skip this for now if you don't mind.